molgenis-data-validation/src/main/java/org/molgenis/data/validation/ExpressionValidator.java

```diff
@@ -57,10 +58,12 @@ boolean resolveBooleanExpression(String expression, Entity entity)
 			return Collections.emptyList();
 		}
 
+		Bindings bindings = jsMagmaScriptEvaluator.createBindings(entity);
```

Comment:
This introduces a dependency on the Java scripting API. Is this desirable?

Reply:
I've switched to evalList method to hide the details.